WhateverGreen/kern_rad.cpp

```diff
@@ -107,6 +107,9 @@ void RAD::init() {
 
 	// Broken drivers can still let us boot in vesa mode
 	forceVesaMode = checkKernelArgument("-radvesa");
+	
+	// Fix codec PID to be spoofed PID if requested
+	forceCodecInfo = checkKernelArgument("-radcodec");
 
```

Comment:
You also need to update initHardwareKextMods. Find `X4000 kext is not included in this list as we need to fix GVA properties for most of its GPUs`. At first we could change the condition to include `&& !forceCodecInfo`, but that will not work if we add a DeviceProperty. Most likely the decision needs to be split:

- Disable incompatible kexts (missing in the target system) in initHardwareKextMods.
- Disable unused kexts (not required by the configuration) in some other method called from processKernel.